config/config_test.go

```diff
@@ -25,7 +25,4 @@ func TestConfig_DataDir(t *testing.T) {
 	config.DataDirParent = "~" + sep + "space-a-mesh"
 	expectedDataDir := filesystem.GetUserHomeDirectory() + sep + "space-a-mesh"
 	assert.Equal(t, expectedDataDir, config.DataDir())
-
-	config.DataDirParent = "~" + sep + "space-a-mesh" + sep // trailing slash should be ignored
```

Comment:
why not test this when OS is ubuntu?

Reply:
This test was already completely removed on `develop`.